milestones/api.py

```diff
@@ -268,25 +284,36 @@ def add_course_content_milestone(course_key, content_key, relationship, mileston
         course_key=course_key,
         content_key=content_key,
         relationship=relationship,
-        milestone=milestone)
+        milestone=milestone,
+        requirement=requirement
```

Comment:
Is there any recommendations around the use of this field by this api? For example, if we call it requirements instead, and recommend using json, it would enable the consumer to easily extend the requirements when and if the time arises. Food for thought.